src/applications/check-in/components/pages/demographics/DemographicsDisplay.jsx

```diff
@@ -77,12 +77,7 @@ export default function DemographicsDisplay({
     <>
       <ConfirmablePage
         header={header || t('is-this-your-current-contact-information')}
-        subtitle={
-          subtitle ||
-          t(
-            'we-can-better-follow-up-with-you-after-your-appointment-when-we-have-your-current-information',
-          )
-        }
+        subtitle={subtitle}
```

Comment:
I have removed the default subtitle here, to prevent it from appearing when there is a telephone appointment. This component is only called from one place per app, and i've moved this specific subtitle to the check-in instance, which should have been the only place where it was being called.